avocado/core/runners/requirement_package.py

```diff
@@ -0,0 +1,130 @@
+import time
```

Comment:
I was thinking about the current support for requirements runners this weekend, and I plan to change it a bit right after we get this code merged.

I concluded that restricting a requirement to a runner that starts with `requirement-` may make the code less flexible when adding support to have any runner, like an `instrumented` test, as a dependency of another test. This will help with tests that depend on other tests on the avocado-vt side, but it needs some adjustments on the current code, and I think it is better to do it in the next PR.